packages/app/cypress/e2e/runner/ct-framework-errors.cy.ts

```diff
@@ -74,7 +74,7 @@ reactVersions.forEach((reactVersion) => {
       })
 
       verify('error on mount', {
-        line: 6,
```

Comment:
Why did these change?

Reply:
the `Errors.cy.jsx` files were changed so the stack trace locations changed